src/pages/library/Showcase.js

```diff
@@ -16,104 +18,337 @@ import * as Style from 'src/libs/style'
         width: 87,
         ...Utils.cond(
           [name.toLowerCase().includes('covid'), () => ({ backgroundImage: `url(${covidBg})` })],
           [variant === 'gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
           () => ({ backgroundImage: `url(${featuredBg})`, opacity: 0.75 })
         )
       }
-    })
+    }),
+    div({ style: { flex: 1, minWidth: 0, padding: '15px 20px', overflow: 'hidden' } }, [
+      div({ style: { display: 'flex' } }, [
+        div({ style: { flex: 1, color: colors.accent(), fontSize: 16, lineHeight: '20px', height: 40, marginBottom: 7 } }, [name]),
+        created && div([Utils.makeStandardDate(created)])
+      ]),
+      div({ style: { lineHeight: '20px', height: 100, whiteSpace: 'pre-wrap', overflow: 'hidden' } }, [description])
+      // h(MarkdownViewer, [description]) // TODO: should we render this as markdown?
+    ])
   ])
 }
 
+// collapsible section for sidebar categories
+const SidebarCollapser = ({ title, isOpened, onClick, children }) => {
+  return div({
+    role: 'group'
+  }, [
+    h(NavItem, {
+      onClick,
+      style: { ...styles.sidebarRow, ...styles.nav.navSection }
+    }, [
+      title,
+      div({ style: { flexGrow: 1 } }),
+      icon(isOpened ? 'angle-up' : 'angle-down', { size: 18, style: { flex: 'none' } })
+    ]),
+    div({
+      style: { flex: 'none' }
+    }, [h(RCollapse, { isOpened }, [children])])
+  ])
+}
+
+const groupByFeaturedTags = workspaces => _.flow([
+  _.map(tag => [tag, _.filter(w => _.includes(tag, w.tags.items), workspaces)]),
+  _.fromPairs
+])(uniqueSidebarTags)
+
+const Sidebar = props => {
+  const { onSectionFilter, onTagFilter, sections, selectedSections, selectedTags, workspacesByTag } = props
+
+  // setup open-ness state for each sidebar section
+  const initialOpenState = _.fromPairs(_.map(section => [section.name, true], sidebarSections))
+  const [openState, setOpenState] = useState(initialOpenState)
+
+  const unionSectionWorkspaces = section => {
+    return _.uniq(_.flatMap(tag => workspacesByTag[tag], section.tags))
+  }
+
+  return div({ style: { display: 'flex', flexDirection: 'column' } }, [
+    _.map(section => {
+      return section.keepCollapsed ?
+        h(Clickable, {
+          key: section.name,
+          onClick: () => onSectionFilter(section),
+          style: { ...styles.sidebarRow, ...styles.nav.navSection }
+        }, [
+          div({ style: { flex: 1 } }, [section.name]),
+          div({ style: { flexGrow: 1 } }),
+          div({
+            style: _.includes(section, selectedSections) ? styles.hilightedPill : styles.pill
+          }, [_.size(unionSectionWorkspaces(section))])
+        ]) :
+        h(SidebarCollapser,
+          {
+            key: section.name,
+            title: section.name,
+            onClick: () => setOpenState(_.update(section.name, open => !open, openState)),
+            isOpened: openState[section.name]
```

Comment:
I don't love the readability of `!(... === false)`, but I can get rid of the special initialization if I reverse it to keep track of collapsed sections instead of open sections. Also, I can turn it into an array and use `_.xor` to toggle the state, just like the filter selections. I prefer thinking of it like that instead of having a sparse object where keys we're looking up might be `undefined` in addition to `false` or `true`.